mne/bem.py

```diff
@@ -980,7 +980,7 @@ def _check_origin(origin, info, coord_frame='head', disp=False):
 @verbose
 def make_watershed_bem(subject, subjects_dir=None, overwrite=False,
                        volume='T1', atlas=False, gcaatlas=False, preflood=None,
-                       show=False, verbose=None):
+                       show=False, verbose=None, force_copy=False):
```

Comment:
also while you're at it can you grep to see where we do also symlinks. I am pretty sure we do it for Flash bem too.